package/python-kiwi-spec-template

```diff
@@ -96,56 +108,72 @@ and cloud systems like Xen, KVM, VMware, EC2 and more.
 %package -n python2-kiwi
 Summary:        KIWI - Appliance Builder Next Generation
 Group:          Development/Languages/Python
-Provides:       python-kiwi = %{version}-%{release}
-Provides:       kiwi-image:tbz
 Provides:       kiwi-image:docker
 Provides:       kiwi-image:iso
-Provides:       kiwi-image:vmx
-Provides:       kiwi-image:pxe
 Provides:       kiwi-image:oem
+Provides:       kiwi-image:pxe
+Provides:       kiwi-image:tbz
+Provides:       kiwi-image:vmx
+Provides:       python-kiwi = %{version}-%{release}
+%if 0%{?fedora} || 0%{?suse_version}
 Recommends:     jing
-Requires:       update-alternatives
+%else
+Requires:       jing
```

Comment:
no requires of jing please, that would mean we require the entire java stack. This should stay as Recommends